fiboa_cli/datasets/ec_si.py

```diff
@@ -0,0 +1,189 @@
+import os
+
+from tempfile import TemporaryDirectory
+from zipfile import ZipFile
+
+from ..util import download_file, log
+from ..convert_utils import convert as convert_
+
+# File to read the data from
+# Can read any tabular data format that GeoPandas can read through read_file()
+# Supported protcols: HTTP(S), GCS, S3, or the local file system
+URI = "https://zenodo.org/records/8229128/files/SI_2021.zip?download=1"
```

Comment:
```suggestion
URI = "https://zenodo.org/records/10118572/files/SI_2021.zip?download=1"
```

Any specific reason why this is running on EuroCrops v9 and not v10?

Reply:
I used the link Chris provided in: https://github.com/fiboa/data/issues/31#issue-2288114517

Reply:
I guess he just copied them randomly, let's go for the latest version.